CHANGES.rst

```diff
@@ -16,9 +16,8 @@ astropy.convolution
 astropy.coordinates
 ^^^^^^^^^^^^^^^^^^^
 
-- The new function :func:`~.coordinates.make_transform_graph_docs` can
-  be used to create a docstring graph from a custom
-  `~.coordinates.TransformGraph` object. [#7135]
+- The new function ``make_transform_graph_docs`` can be used to create a
+  docstring graph from a custom ``TransformGraph`` object. [#7135]
```

Comment:
But now they don't link to the actual documentation pages for the items 😢

Reply:
Yes, we don't do that purposefully. Doc pages can come and go over the years and if we do the changelog will be full of broken links.

Reply:
Would it not be better for it to be full of some broken links for old stuff, and all working links for new stuff, rather than none of the links working at all? Adding the links like I did removes the annoyance of having to copy and paste into the search box and then searching for the new function.

Reply:
I completely agree.

Reply:
To allow broken links means we get future questions on why those links are broken. It is a can of worms.

Reply:
Intersphinx is clever enough that you basically have to *remove* the function for the link to break. And I don't really see that "this link is broken because the function no longer exists" really outweighs the "oh cool a new function that I might want to use, where is the documentation?"

Reply:
We are not beyond removing code. :smirk: Maybe we need an APE on how to properly format change log... :thinking:

Reply:
I remember the amount of headache when we still had the API links there, and foresee some extra work to add those links to the changelog (for many contributors it's not trivial how sphinx work, and what kind of links we want there, etc.) for very very little gain. So I'm 👎 on this, but if you convince enough people I'm OK with that.

Reply:
@Cadair - new cool functionality is  usually detailed in "what's new"...

Reply:
@pllim But *all* the links are broken at the moment, and I am asking why some of them can't be fixed! If the function etc. no longer exists, I think sphinx will just not include a hyperlink instead of providing a broken link.

Reply:
May I suggest we take this to a new issue? I think it's a discussion worth having, there maybe legitimate technical considerations, but I feel that it's worth testing this to see exactly what they are. After all anything thing which makes new functionality in Python more discoverable to people is likely to serve all our best interests.